mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,17 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function getMood(happy) {
+  if (happy == true) {
     return "I am happy";
-  } else {
+  } else if (happy == false){
```

Comment:
since happy can only be either true or false and we want to make sure to enter this block and return "I am not happy" when happy is false, the if else statement should be enough to determine what to return and we shouldn't need a `happy == false` check